eng/scripts/Update-CliVersion.ps1

```diff
@@ -0,0 +1,49 @@
+param(
+    [string] $NewVersion
+)
+
+$CLI_VERSION_FILE = "$PSScriptRoot../../../cli/version.txt"
+. "$PSScriptRoot../../common/scripts/common.ps1"
+
+Set-StrictMode -Version 3
+
+function getVersion {
+    $versionString = Get-Content $CLI_VERSION_FILE
+    return [AzureEngSemanticVersion]::new($versionString)
+}
+
+$version = $NewVersion
+$unreleased = $false
+$replaceLatestEntryTitle = $true
+
+if (!$version) {
+    # Increment after release
+    $version = getVersion
+
+    if ($version.IsPrerelease) {
+        if ($version.HasValidPrereleaseLabel()) {
+            # 0.1.0-beta.1 -> 0.1.0-beta.2
+            $version.PrereleaseNumber++
+        } else {
+            # 0.1.0 -> 0.2.0-beta.1
+            $version.Minor++
+            $version.PrereleaseLabel = 'beta'
+            $version.PrereleaseNumber = 1
+        }
+    } else {
+        # 1.0.0-beta.1 -> 1.0.0-beta.2
```

Comment:
Feels like this example is wrong, since 1.0.0-beta.1 would be a pre-release version right?

Reply:
Indeed this scenario is properly covered on line 24